rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -435,58 +438,53 @@ public void registerBtcTransaction(
     private void registerSvpFundTx(BtcTransaction btcTx) throws IOException {
         logger.info("[registerSvpFundTx] This is an svp fund tx {}", btcTx.getHash());
 
-        processPegoutOrMigration(btcTx); // Need to register the change UTXO
+        registerNewUtxos(btcTx); // Need to register the change UTXO
 
+        // If the SVP validation period is over, SVP related values should be cleared in the next call to updateCollections
+        // In that case, the fundTx will be identified as a regular peg-out tx and processed via #registerPegoutOrMigration
+        // This covers the case when the fundTx is registered between the validation period end and the next call to updateCollections
         if (isSvpOngoing()) {
             updateSvpFundTransactionValues(btcTx);
         }
     }
 
     private void registerSvpSpendTx(BtcTransaction btcTx) throws IOException {
-        logger.info("[registerSvpSpendTx] This is an svp spend tx {}", btcTx.getHash());
-
-        processSvpSpendTransaction(btcTx);
+        logger.info(
+            "[registerSvpSpendTx] This is an svp spend tx {}. SVP was successful, going to commit the proposed federation",
+            btcTx.getHash()
+        );
```

Comment:
Shouldn't we also register the `wtxid` as well for the svpSpendTransaction?

Reply:
Done